src/ucp/core/ucp_context.c

```diff
@@ -52,6 +52,13 @@ static const char *ucp_rndv_modes[] = {
     [UCP_RNDV_MODE_LAST]      = NULL,
 };
 
+static const char *ucp_rndv_frag_mem_types[] = {
```

Comment:
why not resue ucs_memory type_t and ucs_memory_type_names?

Reply:
We don't have a type name for "auto" otherwise we could.